app/templates/admin/createEvent.html

```diff
@@ -252,13 +252,40 @@
     <div class="row justify-content-left mt-2">
           <div class="col-md-6 offset-md-3">
             <div class="text-center">
-              <button type="button" class="btn btn-danger" data-bs-toggle="modal" data-bs-target="#deleteWarning">Delete</button>
+              <button type="button" class="btn btn-danger" data-bs-toggle="modal" data-bs-target="#deleteWarning">Delete Event</button>
+              {% if event and not event.isCanceled %}
+                <button type="button" class="btn btn-warning bordered-element m-2" data-bs-toggle="modal" data-bs-target="#cancelWarning">Cancel Event</button>
+              {% endif %}
               <input type="submit" class="btn btn-primary" value="Save" />
             </div>
           </div>
     </div>
 </form>
-<!-- Modal -->
+<!-- Modal for Cancel Event -->
+<div class="modal fade" id="cancelWarning" tabindex="-1" aria-labelledby="cancelModel" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="cancelModel">Cancel Event</h5>
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <div class="modal-body">
+        <p> Are you sure you want to cancel this event? <br> By proceeding with the cancellation, you understand that this event will no longer take place on the proposed date, but the event information will be kept for potential future use or reference.</p>
```

Comment:
I think I would prefer something like "A canceled event will no longer take place during its scheduled time but the information will be kept for future reference."

Also, the first sentence and second sentence should be in two different `<p>` tags. This change should also be made for the delete events message.

Reply:
Fixed